src/Commands/Admin.py

```diff
@@ -200,6 +201,35 @@ async def forceReport(mentions: str, roles: List[Role], lbChannel: Channel, *arg
         )
 
 
+def multiplier(roles: List[Role], *arg) -> Embed:
+    if (Queue.isBotAdmin(roles)):
+        try:
+            value = float(arg[0])
+        except ValueError:
+            updateMMRMultiplier(1.0)
+            return ErrorEmbed(
+                title="Not a Valid Number",
+                desc="You must enter a positve number."
+            )
+        if (value > 0):
+            multiplier = updateMMRMultiplier(value)
+            return AdminEmbed(
+                title="MMR Multiplied",
+                desc="The MMR gain has been multiplied by a factor of " + str(multiplier)
+            )
+        else:
+            updateMMRMultiplier(1.0)
```

Comment:
Do we need to reset the multiplier to 1 if they mess up the command? Would it be better to just leave whatever it is set to and just return the error?

Reply:
This is what happens whenever the user tries to enter in a value that is lower than 0. It will just set the multiplier to 1 as a safety.

Reply:
I can tell what it does, I'm asking whether we need to reset to 1 as a safety or does it make more sense to just do nothing

Reply:
I feel that it makes more sense to reset to 1 as a safety, this way in case the user makes a mistake, they at least know that it has been defaulted to 1. Odds are, if they're trying to make it negative, then they were trying to mess something up anyways. I think it gives peace of mind for a dev and users of Norm to know that your MMR gain can't be trolled by someone.

Reply:
On that note however, should it be allowed to make the MMR multiplier go less than 1? To be able to gain less than your base amount of MMR?

Reply:
5 * 0.5 = 2.5

Should minimalistic MMR gains be allowed? or should the lowest be 1 times the amount you can gain?

Reply:
Personally, I think it makes more sense to not change anything if there's an error. If the multiplier is 2 and someone tries to set the multiplier to -2, I would expect the multiplier to stay 2 and not reset to 1. But that's me, I'll defer to you and we can always get feedback later if it ever comes up.

As far as doing MMR less than 1, although it doesn't make a whole lot of sense to do that, I don't see a specific reason why we shouldn't allow it. I think blocking negative values and 0 is good, but anything outside of that I think would be good to support.

Reply:
I'm alright to let it return its current value. At least with returning the current value, the user has full control of the multiplier and Norm won't change it.